### PR TITLE
Service Card: fixes for SubTabs on Card

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1401
[Problem] Tabs on service card dosen't work correctly
[Solution]
  - additional css for sub-tabs on card
  - fix for Tabs widget height

### CODE REVIEW

File: examples/mobile/UIComponents/webclip/link-tab/webclip.html [hunk 1]
```diff
@@ -7,10 +7,102 @@
...
+						<li class="ui-li-has-icon ui-li-divider">
+							<div class="ui-li-icon">
+								<img src="images/tw_list_icon_connections.svg"/>
+							</div>
+							<div class="ui-li-text">
+								<span class="ui-li-text-title">
+									Main text 01
+								</span>
+								<span class="ui-li-text-sub">
+									This screen is not settings app.
```
Comment (lslachciak) [inline examples/mobile/UIComponents/webclip/link-tab/webclip.html]: does this text has sense here?

File: examples/mobile/UIComponents/webclip/link-tab/webclip.html [hunk 2]
```diff
@@ -7,10 +7,102 @@
...
+						<li class="ui-li-has-icon">
+							<div class="ui-li-icon">
+								<img src="images/thumbnail_013.jpg"/>
+							</div>
+							<div class="ui-li-text">
+								<span class="ui-li-text-title">
+									Main text 02
+								</span>
+								<span class="ui-li-text-sub">
+									It's focused case of icon+text list
```
Comment (lslachciak) [inline examples/mobile/UIComponents/webclip/link-tab/webclip.html]: focused? what item is focused?

File: examples/mobile/UIComponents/webclip/link-tab/webclip.html [hunk 3]
```diff
@@ -7,10 +7,102 @@
...
+				</section>
+				<section class="ui-content">
+					<ul class="ui-listview ui-content-area">
+						<li class="ui-li-divider">
+							<div class="ui-li-text">
+								<span class="ui-li-text-title">
+									Main text 03
+								</span>
+								<span class="ui-li-text-sub">
+									This is the second line for descriptive phrases. Chunking background color is #fcfcfc.
```
Comment (lslachciak) [inline examples/mobile/UIComponents/webclip/link-tab/webclip.html]: better don't talk about colors in sample texts. Color can be changed by theme change

**Review by lslachciak — COMMENTED** [review]
works fine, but has some comments to sample texts

File: examples/mobile/UIComponents/webclip/link-tab/webclip.html (hunk 3 above)
Comment (DohyungLim) [inline examples/mobile/UIComponents/webclip/link-tab/webclip.html]: I am also the same idea as Lukasz.
You do not need to specify a Color value.

File: examples/mobile/UIComponents/webclip/link-tab/images/tw_list_icon_connections.svg [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (DohyungLim) [inline examples/mobile/UIComponents/webclip/link-tab/images/tw_list_icon_connections.svg:1]: Where is svg used?

**Review by DohyungLim — COMMENTED** [review]
2 comment.

File: examples/mobile/UIComponents/webclip/link-tab/images/tw_list_icon_connections.svg (hunk 4 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/webclip/link-tab/images/tw_list_icon_connections.svg:1]: > Where is svg used?

Yes :) . File ` examples/mobile/UIComponents/webclip/link-tab/webclip.html` line 57.

File: src/js/core/widget/core/Tabs.js [hunk 5]
```diff
@@ -204,9 +204,12 @@
 			 */
 			prototype._initHeight = function () {
 				var self = this,
-					page = ns.widget.Page(self._ui.page);
+					page;
 
-				self.element.style.height = page.getContentHeight() + "px";
+				if (!self.element.style) {
```
Comment (HunseopJeong) [inline src/js/core/widget/core/Tabs.js]: Is it correct condition?
If the 'self.element.style' is 'undefined', I think we cannot set the 'height' value.

File: examples/mobile/UIComponents/webclip/link-tab/webclip.html (hunk 2 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/webclip/link-tab/webclip.html]: Done.

File: examples/mobile/UIComponents/webclip/link-tab/webclip.html (hunk 1 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/webclip/link-tab/webclip.html]: Done.

File: examples/mobile/UIComponents/webclip/link-tab/webclip.html (hunk 3 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/webclip/link-tab/webclip.html]: Done.

Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/webclip/link-tab/webclip.html]: Done.

File: src/js/core/widget/core/Tabs.js (hunk 5 above)
Comment (TomaszLukawskiSam) [reply src/js/core/widget/core/Tabs.js]: Thank you for review. I did mistake in this place. Purpose of this change is remain height if the height was defined by inline styles by developer.

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Pull request merged**